src/meshable_arena.cc

```diff
@@ -750,7 +750,7 @@ void MeshableArena::afterForkChild() {
         }
 #endif
 
-        void *ptr = mmap(remove, sz, HL_MMAP_PROTECTION_MASK, kMapShared | MAP_FIXED, _fd, keepOff * kPageSize);
+        void *ptr = mmap(remove, sz, HL_MMAP_PROTECTION_MASK, kMapShared | MAP_FIXED, newFd, keepOff * kPageSize);
```

Comment:
this is already on master -- @kyoguan after I merge this you should rebase/pull (IDK why github doesn't take care of hiding this)